components/ide/jetbrains/launcher/main.go

```diff
@@ -1067,7 +1067,7 @@ func getProductConfig(config *gitpod.GitpodConfig, alias string) *gitpod.Jetbrai
 }
 
 func linkRemotePlugin(launchCtx *LaunchContext) error {
-	remotePluginDir := launchCtx.backendDir + "/plugins/gitpod-remote"
+	remotePluginDir := launchCtx.configDir + "/plugins/gitpod-remote"
```

Comment:
Isn't this a breaking change for stable?

Reply:
It breaks `2022.3.3`, fixed via pin its launcher version

Reply:
EAP is broken too

Reply:
Double checked, it's broken because `/plugins` folder not exists for repos that don't have JetBrains plugins defined in `.gitpod.yml`, fixed via https://github.com/gitpod-io/gitpod/pull/19588/commits/814d2ad03822e2929ec9c2eb982095949e596167

<img width="759" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/c4e21786-95ec-474d-9a81-0e485111c89e">